playwright/tests/apps.spec.ts

```diff
@@ -1,53 +1,19 @@
+import { URL_LIST } from "@data/url";
 import { AppsPage } from "@pages/appsPage";
-import { AppInstallationPage } from "@pages/appInstallationPage";
+import { MainMenuPage } from "@pages/mainMenuPage";
 import { expect, test } from "@playwright/test";
-import { AppPage } from "@pages/appPageThirdparty";
-import { APPS } from "@data/e2eTestData";
 
-test.use({ storageState: "./playwright/.auth/admin.json" });
-let appsPage: AppsPage;
-let installationPage: AppInstallationPage;
-let appPage: AppPage;
+test.use({ storageState: "playwright/.auth/app.json" });
 
-test("TC: SALEOR_119 User should be able to install and configure app from manifest @e2e", async ({
```

Comment:
Why did you remove this test? It's important to have this covered since we had incident in past.

Reply:
Good catch - I've reverted the unintended changes @szczecha